docs/config-tornjak-agent.md

```diff
@@ -23,7 +23,7 @@ server {
 | Type | Name | Description |
 | ---- | ---- | ----------- |
 | DataStore | [sql]() | Default SQL storage for Tornjak metadata |
-| UserManagement | [keycloak](/docs/plugin_auth_keycloak.md) | Requires JWT Bearer Access Token provided for each request |
+| UserManagement | [keycloak](/docs/plugin_server_auth_keycloak.md) | Requires JWT Bearer Access Token provided for each request |
```

Comment:
Please provide a link to appropriate documentation on how to obtain the Bearer Token, or a link to a future page about that.

Reply:
linked to feature documentation